superset-frontend/src/views/components/MenuRight.tsx

```diff
@@ -23,11 +23,38 @@ import { Link } from 'react-router-dom';
 import Icons from 'src/components/Icons';
 import findPermission from 'src/dashboard/util/findPermission';
 import { useSelector } from 'react-redux';
-import { UserWithPermissionsAndRoles } from 'src/types/bootstrapTypes';
+import {
+  UserWithPermissionsAndRoles,
+  CommonBootstrapData,
+} from 'src/types/bootstrapTypes';
 import LanguagePicker from './LanguagePicker';
 import { NavBarProps, MenuObjectProps } from './Menu';
 
 export const dropdownItems = [
+  {
```

Comment:
if you were to define dropdownItems as `MenuObjectProps[]` here, would typescript then not make you coerce it below on line 164?

Reply:
yep that did it.